.github/workflows/sbom-scan.yml

```diff
@@ -8,7 +8,20 @@ on:
   release:
     types: [published]
 jobs:
+  # Step 1: Upgrade pip before SBOM generation
+  upgrade-pip:
+    name: Upgrade pip before SBOM generation
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/setup-python@v5
+        with:
+          python-version: '3.13'  # use your current workflow Python version
+      - name: Upgrade pip
+        run: python -m pip install --upgrade "pip>=25.2"
```

Comment:
This wouldn't resolve the vulnerability anyway, according to the CVE, it has no patch yet.